drivers/hmis/app/models/hmis/form/form_processor.rb

```diff
@@ -63,8 +63,9 @@ def run!(owner:, user:)
 
         container_processor(container)&.process(field, value)
       rescue StandardError => e
-        Sentry.capture_exception(e)
-        raise $ERROR_INFO, "Error processing field '#{field}': #{e.message}", $ERROR_INFO.backtrace
+        err_with_context = "Error processing field '#{field}': #{e.message}"
+        Sentry.capture_exception(err_with_context)
+        raise $ERROR_INFO, err_with_context, $ERROR_INFO.backtrace
```

Comment:
This second one doesn't get to sentry, idk why. But we want the field context in the message that gets sent to sentry.